engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -96,22 +131,15 @@ protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundE
 
     private final ScriptFinder scriptFinder;
 
-    private final ArrayList<String> scriptImports = new ArrayList<>();
+    private final ImportCustomizer consoleImports = new ImportCustomizer();
```

Comment:
Just to be extra clear, could you add javadoc comments to these saying what they are? (`consoleImports` is imports to add to every command in the interactive console, and `loadedGroovyScriptImports` are imports to apply to any script being ~'source()d'~ loaded?)

Reply:
Correct, added javadoc.